l10n/en/firefox/features/password-manager-2023.ftl

```diff
@@ -19,7 +19,8 @@ password-manager-with-a-free-mozilla-account = With a <a href="{ $fxa }">free {
 # Obsolete string
 # Variables:
 #   $fxa (url) = link to https://www.mozilla.org/firefox/accounts/
-password-manager-with-a-free-firefox-account = With a <a href="{ $fxa }">free { -brand-name-firefox } account</a> you can securely sync your passwords across all your devices.
+password-manager-with-a-free-firefox-account = With a <a href="{ $fxa }">free { -brand-name-firefox-account }</a> you can securely sync your passwords across all your devices.
```

Comment:
So in the original PR I created that added the new strings, I somehow (I honestly don't remember doing this specific change) updated `{ -brand-name-firefox-account }` to `{ -brand-name-firefox } account`. You can see it here: https://github.com/mozilla/bedrock/pull/13690/files#diff-29a156658b629619541921aa462c94521caa4f044fbe03d445616b3d3f65e318L17

So the L10N team wanted me to change it back to how it was since it was becoming an obsolete string anyway